scripts/pocs-shell.py

```diff
@@ -190,7 +190,17 @@ def do_reset_pocs(self, *arg):
 
         Does NOT park the mount, nor execute power_down.
         """
-        self.pocs = None
+        if self.pocs.observatory.mount.is_parked is False:
+            print("ATTENTION: The mount is not in the parked position and reset_pocs will not move the mount to park position!")
+            response = input("The unit could be damaged if it is not parked before sunrise. Execute reset_pocs anyway?[y/n]")
+            if response == "y" or response == "yes":
```

Comment:
A slightly cleaner way to do something like this is to force their input into a format that is easier to check:

```suggestion
            if response.lower().startswith('y'):
```